problems/arrayMethods.js

```diff
@@ -5,7 +5,15 @@
  * @returns {number} Sum of all numbers
  */
 
-const sumArray = () => {};
+const sumArray = (nums) => {
+  let sum = 0
+  nums.forEach((num) => {
+    nums.push(sum += num)
```

Comment:
Even though this is passing the test, this is not the right way to solve this.

You're pushing into the parameter "nums" which is not what we want to do. We don't want to directly mutate the parameter, we also don't need to push in this scenario.

We want to just write sum+=num on line 11